service/src/main/java/com/zebrunner/reporting/service/feign/FeignClientInterceptor.java

```diff
@@ -0,0 +1,25 @@
+package com.zebrunner.reporting.service.feign;
+
+import com.zebrunner.reporting.domain.dto.auth.AuthenticatedUser;
+import feign.RequestInterceptor;
+import feign.RequestTemplate;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.Component;
+
+@Component
+public class FeignClientInterceptor implements RequestInterceptor {
```

Comment:
Is it meant to be called after a library it uses?

Reply:
RequestInterceptor used in this class also belongs to the feign library, I've left this naming to mark that this interceptor serves only to the library request purpose